e2eAIOK/common/trainer/utils.py

```diff
@@ -173,29 +183,25 @@ def log_every(self, iterable, print_freq, header=None):
         print('{} Total time: {} ({:.4f} s / it)'.format(
             header, total_time_str, total_time / len(iterable)))
 
-
-def decode_arch_tuple(arch_tuple):
-    arch_tuple = ast.literal_eval(arch_tuple)
-    depth = int(arch_tuple[0])
-    mlp_ratio = [float(x) for x in (arch_tuple[1:depth+1])]
-    num_heads = [int(x) for x in (arch_tuple[depth + 1: 2 * depth + 1])]
-    embed_dim = int(arch_tuple[-1])
-    return depth, mlp_ratio, num_heads, embed_dim
-
-def create_operation(model, cfg):
-    if cfg.criterion == "CrossEntropyLoss":
-        criterion = torch.nn.CrossEntropyLoss()
+    
+def create_optimizer(model, cfg):
     if cfg.optimizer == "SGD":
         optimizer = torch.optim.SGD(model.parameters(), lr=cfg.learning_rate,
                 momentum=cfg.momentum, weight_decay=cfg.weight_decay)
-    if cfg.lr_scheduler == "CosineAnnealingLR":
-        lr_scheduler = torch.optim.lr_scheduler.CosineAnnealingLR(optimizer, T_max=cfg.train_epochs)
-    all_operations = {'criterion':criterion,'optimizer':optimizer,'lr_scheduler':lr_scheduler}
-    return all_operations
+    return optimizer
 
 def create_metric(output, target, cfg):
     if cfg.eval_metric == "accuracy":
         acc1, acc5 = accuracy(output, target, topk=(1, 5))
         metric = {'acc1':acc1,'acc5':acc5}
-        return metric
-    
```

Comment:
1. These functions are duplicated in torch_trainer.py, maybe we can only keep one
2. Suggest move utils.py to util file

Reply:
move functions from torch_trainer to utils.py and move utils.py to utils folder